### ISSUE TITLE
I'd like to update rules with the Thinkerbell adapter

### ISSUE THREAD

**Issue by Yoric** [fxbox/foxbox#325]
STR:

```
curl -X PUT http://localhost:3000/api/v1/channels/set -d '{"select":{"kind":"AddThinkerbellRule"},"value":{"ThinkerbellRule":{"name":"Hello, Thinkerbell","source":"{\"rules\":[{\"conditions\":[{\"source\":[{\"id\":\"OpenZWave/72057594126794752 (Sensor)\"}],\"kind\":\"OpenClosed\",\"range\":{\"Eq\":{\"OpenClosed\":\"Open\"}}}],\"execute\":[{\"destination\":[{\"kind\":\"Log\"}],\"kind\":\"Log\",\"value\":{\"String\":\"Closed\"}}]}]}"}}}'
```

(do it twice)

Expected results: the second rule should replace the first one.

Actual results:

```
[2016-04-08 16:09:46.655] ERROR Unable to add Thinkerbell Rule Service: InternalError(DuplicateService(Id { id: Atom('thinkerbell/Hello, Thinkerbell' type=dynamic), phantom: Phantom { phantom: PhantomData } }))
```

This is pretty annoying for demo development, debugging, etc. @mcav, could you take a look at this? 

Cc @ferjm

**Comment by mcav** [comment]
Yeah, I'll take this.

**Comment by mcav** [comment]
Resolved in #334.

**Comment by Yoric** [comment]
Now I get the following response: `{"thinkerbell-add-rule":{"Error":{"InternalError":{"GenericError":"RunError(StartStopError(AlreadyRunning))"}}}}`.
